ui/src/components/modals/create-bedset-modal.tsx

```diff
@@ -0,0 +1,134 @@
+import { useCopyToClipboard } from '@uidotdev/usehooks';
+import { useState } from 'react';
+import { Modal } from 'react-bootstrap';
+import Markdown from 'react-markdown';
+import { generateBEDsetPEPMd, generateBEDsetPEPDownloadRaw } from '../../utils';
+import { useBedCart } from '../../contexts/bedcart-context';
+import rehypeHighlight from 'rehype-highlight';
+import axios from 'axios';
+
+type Props = {
+  show: boolean;
+  setShow: (show: boolean) => void;
+};
+
+const API_BASE = import.meta.env.VITE_API_BASE || '';
+const API_ENDPOINT = `${API_BASE}/bedset/create/`;
+
+export const generateBEDsetCreationDescription = () => {
+  const text = `
+  **To create a new BEDset:**
+
+  1. Create PEP in [PEPhub](https://pephub.databio.org/), by copying the text below, and pasting it into the sample table.
+  The name of the PEP project will be used as the name and identifier for the BEDset.
+  2. Add source, author, and other metadata to config file. e.g. 
+  \`\`\`json
+  'author': "BEDbase team",
+  'source': "BEDbase",
+  \`\`\`
+  3. Use 'Submit PEP' form below or BEDbase API ([${API_BASE}](${API_BASE}/docs#/bedset/create_bedset_v1_bedset_create__post)) and 
+  create a new BEDset by providing the registry path in the Body of the request. (Registry path can be copied from the PEPhub):
+  \`\`\`json
+  {
+    "registry_path": "namespace/name:tag"
+  }
+  \`\`\`
+  **Note**: We currently only support PEPs from the bedbase team. If you want to create new bedset, please create an issue: [https://github.com/databio/bedbase](https://github.com/databio/bedbase/issues)
+
+  `;
+  return text;
+};
+
+export const CreateBedSetModal = (props: Props) => {
+  const { show, setShow } = props;
+  const { cart } = useBedCart();
+  const [, copyToClipboard] = useCopyToClipboard();
+  const [copied, setCopied] = useState(false);
+
+  const [inputValue, setInputValue] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleInputRegistryPathChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.target.value);
+  };
+
+  const handleSubmit = async () => {
+    try {
+      await axios.post(API_ENDPOINT, { registry_path: inputValue });
+      setMessage('Successfully created BEDset!');
+    } catch (error) {
+      const err = error as Error & { response?: { data?: { detail?: string } } };
+      const errorMessage = err.response?.data?.detail || err.message;
+      setMessage(`! Unable to create BEDset. ${errorMessage}`);
+    }
+  };
```

Comment:
Bigger discussion than a PR review, but we probably eventually want to convert this to a `useMutation` implementation. Also, can anyone submit a bedset?